cmd/generate/config/rules/nuget.go

```diff
@@ -0,0 +1,29 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"regexp"
+
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func NugetConfigPassword() *config.Rule {
+	r := config.Rule{
+		Description: "Identified a password within a Nuget config file, potentially compromising package management access.",
+		RuleID:      "nuget-config-password",
+		Regex:       regexp.MustCompile(`(?i)(<add key=\"(ClearTextPassword)|(Password)\"\s*)(value=\"([A-Za-z0-9]+)\"\s*)(/>)`),
+		Path:        regexp.MustCompile(`(?i)nuget\.config$`),
+		SecretGroup: 5,
```

Comment:
This should have keywords for efficiency.

e.g.,
```suggestion
		Keywords: []string{"<add"},
```

Reply:
nice thanks

Reply:
@bufferoverflow `<add key=` would be even better

Reply:
Do we know if it's possible for there to be deviations in the format? E.g., multiple spaces or tabs between `<add` and `key=`?

Reply:
@zricethezav I changed accordingly.


> Do we know if it's possible for there to be deviations in the format? E.g., multiple spaces or tabs between <add and key=?

@rgmz not sure if this is or could be the case